src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,220 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.OrderDao;
+import mate.academy.exceptions.DataProcessingException;
+import mate.academy.lb.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.Product;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+
+    @Override
+    public Order create(Order order) {
+        String insertOrder = "INSERT INTO orders (user_id) VALUES (?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement insertOrderQueryStatement
+                        = connection.prepareStatement(insertOrder,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            insertOrderQueryStatement.setLong(1, order.getUserId());
+            insertOrderQueryStatement.executeUpdate();
+            ResultSet resultSet = insertOrderQueryStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long orderId = resultSet.getLong(1);
+                order.setId(orderId);
+                insertOrderQueryStatement.close();
+                insertOrdersProducts(order, connection);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Creating order " + order
+                    + " is failed", e);
+        }
+        return order;
+    }
+
+    @Override
+    public Order update(Order order) {
+        String updateOrder = "UPDATE orders SET user_id = ? WHERE order_id = ? AND deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateOrderStatement
+                        = connection.prepareStatement(updateOrder)) {
+            updateOrderStatement.setLong(1, order.getUserId());
+            updateOrderStatement.setLong(2, order.getId());
+            if (updateOrderStatement.executeUpdate() == 1) {
```

Comment:
if user_id has not been changed will the update method return 1? If yes then it's okay, if not then reconsider this solution